src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -20,17 +20,17 @@
             + "\t    | | |_| | || (_) | |  | | (_| | | | |  | | (_) | (_| |  __/_|\n"
             + "\t    |_|\\__,_|\\__\\___/|_|  |_|\\__,_|_| |_|  |_|\\___/ \\__,_|\\___(_)\n";
 
-    public static final String[] CIRCUIT_INSTRUCTIONS = {"Lets get started! First we will choose a template to work "
-            + "with. In the app you can choose from rc, rl, lc and r templates." + System.lineSeparator() + "\tHowever,"
-            + " for this tutorial, we will use a Resistor-Capacitor template." + System.lineSeparator()
+    public static final String[] CIRCUIT_INSTRUCTIONS = {"Let's get started! First we will choose a template to work "
+            + "with. In the app you can choose from rc, rl, lc, and r templates." + System.lineSeparator() + "\tHowever"
```

Comment:
Looking at #162 maybe a newline could be added before "In the app"?

Reply:
Made this change in my system on a branch i'm currently working on for the PPP